pkgs/development/python-modules/gmsh/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, buildPythonPackage
+, substituteAll
+, pkgs
```

Comment:
Please do not input pkgs.

Reply:
I have not gotten around to updating this since figuring out the right way to do it [here](https://github.com/NixOS/nixpkgs/pull/103554).